Solutions/CiscoDuoSecurity/Analytic Rules/CiscoDuoAdminMFAFailures.yaml

```diff
@@ -0,0 +1,31 @@
+id: e46c5588-e643-4a60-a008-5ba9a4c84328
+name: Cisco Duo - Multiple admin 2FA failures
+description: |
+  'Detects when multiple admin 2FA failures occurs.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: CiscoDuoSecurity
+    dataTypes:
+      - CiscoDuo
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1078
+query: |
+  let threshold = 10;
+  CiscoDuo
+  | where DvcAction =~ "admin_2fa_error"
+  | summarize count() by DstUserName, bin(TimeGenerated, 10m)

```

Comment:
Might be a good idea to run this query and check. After the summarize statement it seems like the result will have only three columns DstUserName, TimeGenerated and count_. You cannot extend IPCustomEntity and use it in Entity mapping if the column is not there. If there are other columns relevant to the result, please out put them as well when doing summarize.

Reply:
fixed